src/ansys/additive/core/porosity.py

```diff
@@ -129,39 +112,39 @@ def material(self, value):
     def size_x(self):
         """Size (m) of the simulated sample in the x dimension.
 
-        Valid values are from 1e-3 to 1e-2 m (1 to 10 mm).
+        Valid values are from :obj:`MIN_SAMPLE_SIZE` to :obj:`MAX_SAMPLE_SIZE` meters.
         """
         return self._size_x
 
     @size_x.setter
     def size_x(self, value):
-        self.__validate_range(value, self.__MIN_SAMPLE_SIZE, self.__MAX_SAMPLE_SIZE, "size_x")
+        self.__validate_range(value, self.MIN_SAMPLE_SIZE, self.MAX_SAMPLE_SIZE, "size_x")
         self._size_x = value
 
     @property
     def size_y(self):
         """Size (m) of the simulated sample in the y dimension.
 
-        Valid values are from 1e-3 to 1e-2 m (1 to 10 mm).
+        Valid values are from :obj:`MIN_SAMPLE_SIZE` to :obj:`MAX_SAMPLE_SIZE` meters.
         """
         return self._size_y
 
     @size_y.setter
     def size_y(self, value):
-        self.__validate_range(value, self.__MIN_SAMPLE_SIZE, self.__MAX_SAMPLE_SIZE, "size_y")
+        self.__validate_range(value, self.MIN_SAMPLE_SIZE, self.MAX_SAMPLE_SIZE, "size_y")
         self._size_y = value
 
     @property
     def size_z(self):
         """Size (m) of the simulated sample in the z dimension.
 
-        Valid values are from 1e-3 to 1e-2 m (1 to 10 mm).
+        Valid values are from :obj:`MIN_SAMPLE_SIZE` to :obj:`MAX_SAMPLE_SIZE` meters.
```

Comment:
```suggestion
        Valid values are from the :obj:`MIN_SAMPLE_SIZE` value to the
        :obj:`MAX_SAMPLE_SIZE` value.
```
Same question here about meters.